src/plugins/embeddable/public/lib/embeddables/default_embeddable_factory_provider.ts

```diff
@@ -37,11 +37,11 @@ export const defaultEmbeddableFactoryProvider = <
     getExplicitInput: def.getExplicitInput
       ? def.getExplicitInput.bind(def)
       : () => Promise.resolve({}),
-    createFromSavedObject:
-      def.createFromSavedObject ??
-      ((savedObjectId: string, input: Partial<I>, parent?: IContainer) => {
-        throw new Error(`Creation from saved object not supported by type ${def.type}`);
-      }),
+    createFromSavedObject: def.createFromSavedObject
```

Comment:
I'm not quite sure why this change is required, why the bind, and why a ternary here instead of nullish coalescing?

Reply:
This wasn't bound properly and was causing an error when creating visualize embeddable from saved object.